src/app/[locale]/dashboard/@board/Grid/Row/Row.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react';
+
+import styles from './Row.module.css';
+
+type Props = {
+  children?: ReactNode;
+};
+
+function Row({ children }: Props) {
+  return <div className={styles.root}>{children}</div>;
+}
```

Comment:
There is a react type you can use in these cases called `React.PropsWithChildren`, you can replace all of these with just:
```suggestion

function Row({ children }: React.PropsWithChildren) {
  return <div className={styles.root}>{children}</div>;
}
```